for_devs/local-net

```diff
@@ -158,11 +158,13 @@ def config(binary, boot_nodes, rpc_nodes):
                 del genesis_balances[genesis_balances_faucet_address]
                 genesis_balances[faucet_public_address] = genesis_balances_faucet_value
 
+                genesis_state = genesis_state[list(genesis_state.keys())[0]]
+                dpos_config_balance = genesis_state[list(genesis_state.keys())[0]]
```

Comment:
what is the difference between `genesis_state`  and `dpos_config_balance` ?

Reply:
If you look closely in the first line we are overwriting `genesis_state`, so the values are different. 
First one is mapping address -> balance, but second one is just balance to save value from config. 
That done to save value from the config, but change addresses. Before it was just setting to eligibility_balance_threshold